src/Viewer/renderers/mathInput.jsx

```diff
@@ -346,6 +350,14 @@ export default function MathInput(props) {
               substituteTextarea: function () {
                 textareaRef.current = document.createElement('textarea');
                 textareaRef.current.disabled = SVs.disabled;
+                textareaRef.current.addEventListener("focusout", (e) => {
+                  let keyboard = document.getElementById("keyboard");
```

Comment:
Thinking about this a little more, we probably don't want this code to throw an exception if we happen to ever end up with a page that has a MathInput and no keyboard (we likely always want the keyboard to always appear with math inputs, but a silent exception here isn't the right way to enforce that), I would change the if to something like `keyboard && keyboard.contains...`

Reply:
Right, it turns out this crashes the test page and the cypress test page, as they don't have the keyboard.  I'll make a new PR changing it to `keyboard?.contains`.